### DISCUSSION TITLE
could not acquire lock-file! abort..

### DISCUSSION THREAD

**Discussion by Mintxshi** [FelixKratz/SketchyBar#235] [Q&A] [answered]
when i run sketchybar, it gives me this error
```
sketchybar: could not acquire lock-file! abort..
```

**Comment by FelixKratz** [comment, marked as answer]
This is usually because sketchybar is already running. Run:
```bash
brew services stop sketchybar
killall sketchybar
```
If you are certain that it is not already running you could try to delete the lock-file manually:
```bash
rm -rf /tmp/sketchybar_$USER.lock
```

If you start sketchybar for the commandline now it should work as expected.

**Reply by imyanice** [reply]
Hi, I did what you said but I'm getting all these weird errors:
`~ via ⬢ v18.8.0 took 2s 
➜ sketchybar
sketchybar configuration loaded..
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/clock.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/space.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/battery.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/battery.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/clock.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/window_title.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/battery.sh: No such file or directory
sh: /Users/yanjobs/.config/sketchybar/plugins/clock.sh: No such file or directory`

**Reply by FelixKratz** [reply]
What is weird about these errors? They tell you exactly what is wrong. You are missing the specified files in the specified directory.

**Reply by imyanice** [reply]
Me is dumb, sorry
